### ISSUE TITLE
Xray Integration as a Service

### ISSUE THREAD

**Issue by psubrambe** [webdriverio/webdriverio#5958]
**Is your feature request related to a problem? Please describe.**
There are lot of way is available after execution reports uploaded to Xray, but we need to code in different formats like few creating a stage Jenkins to upload the reports.

**Describe the solution you'd like**
Xray Integration (uploading execution status of scripts) as a  wdio service 

**Describe alternatives you've considered**
Basically cucumber.json files will have the test scripts execution status just need to integrate with Xray Apis 

**Additional context**
Same as like Zafira Listener Service

**Comment by christian-bromann** [comment]
@psubrambe sounds like a reasonable suggestion and a great wdio service. Nothing necessarily we need to bring into core though. Would love to add such a service into the config wizard. Let me know how it goes and if you have any questions.

**Comment by psubrambe** [comment]
@christian-bromann : Config wizard means I am not sure could you please guide me on it.
